server/model/traqMessage.go

```diff
@@ -0,0 +1,69 @@
+package model
+
+import "strings"
+
+func TraqMessageProcessor(messageList MessageList) (SendList, error) {
+	words := []Words{}
+	err := db.Select(&words, "SELECT word, bot_notification, me_notification, trap_id FROM words")
+	if err != nil {
+		return nil, err
+	}
+	var sendList SendList
+	for _, message := range messageList {
+		for _, word := range words {
+			if strings.Contains(message.Content, word.Word) {
+				if message.UserId == word.UserId {
+					if !word.IncludeMe {
+						continue
+					}
+					if !word.IncludeBot {
+						if IsBot(message.UserId) {
+							continue
+						}
+					}
+					// 通知する内容を追加
+					sendList = append(sendList, Send{
+						word:      word.Word,
+						userId:    word.UserId,
+						messageId: message.Id,
+					})
+				}
+			}
```

Comment:
メッセージ投稿者と単語登録者が同一のときしか処理できてなさそう
多分こういうミスでは
```suggestion
				if message.UserId == word.UserId {
					if !word.IncludeMe {
						continue
					}
				}
				if !word.IncludeBot {
					if IsBot(message.UserId) {
						continue
					}
				}
				// 通知する内容を追加
				sendList = append(sendList, Send{
					word:      word.Word,
					userId:    word.UserId,
					messageId: message.Id,
				})
```